blog/app/models/article.rb

```diff
@@ -0,0 +1,4 @@
+class Article < ActiveRecord::Base
+	validates :title, presence: true,
+		length: {minimum: 5}				
```

Comment:
usually, validations stay on one line and get cut only if the `character by line` limit is met.